docs/services/frame-system/_index.md

```diff
@@ -0,0 +1,173 @@
+
+Navigate to the **CONFIG** tab on your robot's page in [the Viam app](https://app.viam.com), select the **Builder** mode, scroll to a component's card, and click **Add Frame**:
+
+![add reference frame pane](img/frame_card.png)
+
+{{% /tab %}}
+{{% tab name="JSON Template" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "components": [
+    {
+      "name": <"your_component_name_1">,
+      "type": <"your_component_type_1">,
+      "model": <"your_component_model_1">,
+      "attributes": { ... },
+      "depends_on": [],
+      "frame": {
+        "parent": <"world">,
+        // default
+        "translation": {
+          "y": 0,
+          "z": 0,
+          "x": 0
+        },
+        // default
+        "orientation": {
+          "type": <"ov_degrees">,
+          "value": {
+            "x": 0,
+            "y": 0,
+            "z": 1,
+            "th": 0
+          }
+        }
+      }
+    },
+    {
+      "depends_on": [],
+      "name": <"your_component_name_2">,
+      "type": <"your_component_type_2">,
+      "model": <"your_component_model_2">,
+      "attributes": { ... },
+      "frame": {
+        "parent": <"your_component_name_1">,
+        // default
+        "translation": {
+          "x": 0,
+          "y": 0,
+          "z": 0
+        },
+        // default
+        "orientation": {
+          "type": <"ov_degrees">,
+          "value": {
+            "x": 0,
+            "y": 0,
+            "z": 1,
+            "th": 0
+          }
+        }
+      }
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+Configure the reference frame as follows:
+
+| Parameter | Inclusion | Required |
+| --------- | ----------- | ----- |
+| `Parent`  | **Required** | Default: `world`. The name of the reference frame you want to act as the parent of this frame. |
+| `Translation` | **Required** | Default: `(0, 0, 0)`. The coordinates that the origin of this component's reference frame has within its parent reference frame. |
+| `Orientation`  | **Required** | Default: `(0, 0, 1), 0`. The [orientation vector](/internals/orientation-vector/) that yields the axes of the component's reference frame when applied as a rotation to the axes of the parent reference frame. <br> Types: `Orientation Vector Degrees`, `Orientation Vector Radians`, and `Quaternion`. |
+| `Geometry`  | **Required** | Default: `none`. Collision geometries for defining bounds in the environment of the robot. <br> Types: `Sphere`, `Box`, and `Capsule`. |
+
+{{% alert title="Caution" color="caution" %}}
+
+`Types` are offered in `Orientation` for ease of configuration, but the [orientation vector](/internals/orientation-vector/) type that is always stored and returned by the Frame System is `"Orientation Vector Radians"`.
+`"Degrees"` and `"Quaternion"` will be converted to `"Radians"`.
+
+{{% /alert %}}
+
+{{% alert title="Tip" color="tip" %}}
+
+Viam's coordinate system considers `+X` to be forward, `+Y` to be left, and `+Z` to be up.
```

Comment:
[suggest] Make mention of the "right hand rule"? https://en.wikipedia.org/wiki/Right-hand_rule

Reply:
done, LMK if you have any suggestions